src/Components/Calculator.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+
+class Calculator extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { };
+  }
+
+  render() {
+    return (
+      <ul className="items-container">
+        <li className="flex-center result">
+          <input type="text" />
+        </li>
+        <li className="flex-center keyItem">
+          <button className="btn" type="button">AC</button>
+          <button className="btn" type="button">+/-</button>
+          <button className="btn" type="button">&#x25;</button>
+          <button className="btn" type="button">&#xf7;</button>
+        </li>
+        <li className="flex-center keyItem">
+          <button className="btn" type="button">7</button>
+          <button className="btn" type="button">8</button>
+          <button className="btn" type="button">9</button>
+          <button className="btn" type="button">x</button>
+        </li>
+        <li className="flex-center keyItem">
+          <button className="btn" type="button">4</button>
+          <button className="btn" type="button">5</button>
+          <button className="btn" type="button">6</button>
+          <button className="btn" type="button">-</button>
+        </li>
+        <li className="flex-center keyItem">
+          <button className="btn" type="button">1</button>
+          <button className="btn" type="button">2</button>
+          <button className="btn" type="button">3</button>
+          <button className="btn" type="button">+</button>
+        </li>
+        <li className="flex-center keyItem">
+          <button className="btn" type="button">0</button>
+          <button className="btn" type="button">.</button>
+          <button className="btn" type="button">=</button>
+        </li>
+      </ul>
+    );
+  }
+}
```

Comment:
- [ ] Kindly make sure to use the functional components in order to meet the project requirement. :wink: 
    
   ![image](https://user-images.githubusercontent.com/87197412/218887662-67a68405-5f23-4545-919b-c90aad693a02.png)